packages/manager-react-components/src/hooks/services/hooks/useServiceDetails.ts

```diff
@@ -0,0 +1,35 @@
+import { ApiError, ApiResponse } from '@ovh-ux/manager-core-api';
+import { useQuery, useQueryClient } from '@tanstack/react-query';
+import {
+  getResourceServiceId,
+  getResourceServiceIdQueryKey,
+  getServiceDetails,
+} from '../api';
+import { ServiceDetails } from '../services.type';
+
+export const serviceDetailsQueryKey = ['service-details'];
```

Comment:
**nitpick:** Query key should also include serviceId otherwise can have a wrong result is query is made with a different service due to caching.
```suggestion
export const serviceDetailsQueryKey = ['service-details'];
```

Reply:
```suggestion
export const serviceDetailsQueryKey = (resourceName) => ['service-details', resourceName];
```

Reply:
Done